stores-api/governance-api/src/main/java/com/bloxbean/cardano/yaci/store/api/governance/controller/LocalCommitteeController.java

```diff
@@ -0,0 +1,37 @@
+package com.bloxbean.cardano.yaci.store.api.governance.controller;
+
+import com.bloxbean.cardano.yaci.store.api.governance.dto.CommitteeDto;
+import com.bloxbean.cardano.yaci.store.api.governance.dto.LocalCommitteeDto;
+import com.bloxbean.cardano.yaci.store.api.governance.service.LocalCommitteeService;
+import com.bloxbean.cardano.yaci.store.governance.service.LocalGovStateService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("${apiPrefix}/local/committees")
```

Comment:
Should we also add governance prefix?

${apiPrefix}/governance/local/committees

Reply:
I fixed this